crates/sncast/src/helpers/interactive.rs

```diff
@@ -0,0 +1,270 @@
+            [snfoundry]
+            key = 2137
+
+            [sncast.default]
+            account = "testnet"
+            url = "https://localhost:5050"
+
+            # comment
+
+            [sncast.testnet]
+            account = "testnet-account"        # comment
+            url = "https://swmansion.com/"
+        "#};
+
+        let mut toml_doc = original.parse::<DocumentMut>().unwrap();
+
+        update_config(&mut toml_doc, "default", "account", "testnet");
+
+        assert_eq!(toml_doc.to_string(), expected);
+    }
+
+    #[test]
+    fn test_create_key() {
+        let original = formatdoc! {r#"
+            [snfoundry]
+            key = 2137
+
+            [sncast.default]
+            url = "https://localhost:5050"
+
+            [sncast.testnet]
+            account = "testnet-account"        # comment
+            url = "https://swmansion.com/"
+        "#};
+
+        let expected = formatdoc! {r#"
+            [snfoundry]
+            key = 2137
+
+            [sncast.default]
+            url = "https://localhost:5050"
+            account = "testnet"
+
+            [sncast.testnet]
+            account = "testnet-account"        # comment
+            url = "https://swmansion.com/"
+        "#};
+
+        let mut toml_doc = original.parse::<DocumentMut>().unwrap();
+
+        update_config(&mut toml_doc, "default", "account", "testnet");
+
+        assert_eq!(toml_doc.to_string(), expected);
+    }
+
+    #[test]
+    fn test_create_table_key() {
+        let original = formatdoc! {r#"
+            [snfoundry]
+            key = 2137
+
+            [sncast.testnet]
+            account = "testnet-account"        # comment
+            url = "https://swmansion.com/"
+        "#};
+
+        let expected = formatdoc! {r#"
+            [snfoundry]
+            key = 2137
+
+            [sncast.testnet]
+            account = "testnet-account"        # comment
+            url = "https://swmansion.com/"
+
+            [sncast.default]
+            account = "testnet"
+        "#};
+
+        let mut toml_doc = original.parse::<DocumentMut>().unwrap();
+
+        update_config(&mut toml_doc, "default", "account", "testnet");
+
+        assert_eq!(toml_doc.to_string(), expected);
+    }
+
+    #[test]
+    fn test_create_table() {
+        let original = formatdoc! {r#"
+            [snfoundry]
+            key = 2137
+        "#};
+
+        let expected = formatdoc! {
+            r#"
+            [snfoundry]
+            key = 2137
+
+            [sncast]
+
+            [sncast.default]
+            account = "testnet"
+        "#};
+
+        let mut toml_doc = original.parse::<DocumentMut>().unwrap();
+
+        update_config(&mut toml_doc, "default", "account", "testnet");
+
+        assert_eq!(toml_doc.to_string(), expected);
+    }
+
+    #[test]
+    fn test_create_table_empty_file() {
+        let original = "";
+
+        let expected = formatdoc! {
+            r#"
+            [sncast]
```

Comment:
I am unsure if it is possible to avoid the creation of this artifact.  It's probably due to how `edit_toml` work

Reply:
And it doesn't change anything